sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -541,6 +553,182 @@ def set_read_only(
         except binascii.Error:
             raise binascii.Error("Connection string secret has incorrect padding")
 
+    @distributed_trace
+    def begin_create_snapshot(
+        self,
+        name: str,
+        filters: List[Dict[str, str]],
+        *,
+        composition_type: Optional[Literal["key", "key_label"]] = None,
+        retention_period: Optional[int] = None,
+        tags: Optional[Dict[str, str]] = None,
+        **kwargs
+    ) -> LROPoller[Snapshot]:
+        """Create a snapshot of the configuration settings.
+
+        :param name: The name of the snapshot to create.
+        :type name: str
+        :param filters: A list of filters used to filter the configuration settings by key field and label field
+            included in the snapshot. Each filter should be in format {"key": <value>, "label": <value>},
+            and key field value is required.
+        :type filters: list[dict[str, str]]
+        :keyword str composition_type: The composition type describes how the key-values
+            within the snapshot are composed. Known values are: "key" and "key_label". The "key" composition type
+            ensures there are no two key-values containing the same key. The 'key_label' composition type ensures
+            there are no two key-values containing the same key and label.
+        :keyword int retention_period: The amount of time, in seconds, that a snapshot will remain in the
+            archived state before expiring. This property is only writable during the creation of a
+            snapshot. If not specified, will set to 2592000(30 days). If specified, should be
+            in range 3600(1 hour) to 7776000(90 days).
+        :keyword dict[str, str] tags: The tags of the snapshot.
+        :return: A poller for create snapshot operation. Call `result()` on this object to wait for the
+            operation to complete and get the created snapshot.
+        :rtype: :class:`~azure.core.polling.LROPoller[~azure.appconfiguration.Snapshot]`
```

Comment:
We shouldn't use the `:class:` tag within `:rtype:`
```suggestion
        :rtype: ~azure.core.polling.LROPoller[~azure.appconfiguration.Snapshot]
```